kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -216,6 +169,23 @@
       closeCreateUserModal() {
         this.creatingUser = false;
       },
+      matchesText(user) {
```

Comment:
I agree it's a bit odd to have a helper methods defined here which are only ever used inside `visibleUsers`.

You could define `matchesText` and `matchesRole` as locally scoped, named functions inside `visibleUsers` using syntax like `const matchesText = () => {...}`. That gives some of the same benefits of breaking up the logic, while keeping it clear that the functions aren't really meant to be re-used.

Another option might be to keep them under `methods` but name them `this._matchesText` and `this._matchesRole`. Underscores are sometimes used to denote more "private" methods and attributes, but it's not a convention we've been using in _.vue_ files so I'm not sure how appropriate it is here.